exopite-simple-options/fields/content.php

```diff
@@ -24,7 +24,7 @@ public function output() {
 				if ( is_callable( $callback['function'] ) ) {
 
 					$args    = ( isset( $callback['args'] ) ) ? $callback['args'] : '';
-					$content = call_user_func( $callback['function'], $args );
+					$content .= call_user_func( $callback['function'], $args );
```

Comment:
In fact, it should overwrite, but I can live with this solution too :)

Reply:
I needed to have the 'callback' complete the 'content',  couldn't find an easier solution :)  
Let me know if you like to add this behavior as an option